src/main/java/yapper/Yapper.java

```diff
@@ -0,0 +1,66 @@
+package yapper;
+
+import java.util.Scanner;
+
+import yapper.datastorage.DataStorage;
+import yapper.parser.Parser;
+import yapper.tasktypes.TaskList;
+import yapper.ui.UI;
+
+
+/**
+ * The main class for the Yapper chatbot, which manages task storage, user interactions, and command execution.
+ */
+public class Yapper {
+    private Scanner scanner;
+    private TaskList taskList;
+    private DataStorage dataStorage;
+    private String botName = "Yapper";
```

Comment:
Should this be a constant instead?
```suggestion
    private static final String BOT_NAME = "Yapper";
```